apps/storybook/src/overlays/DropdownMenu.stories.tsx

```diff
@@ -220,3 +230,58 @@ export const TriggerWidth: Story = {
     </Flex>
   ),
 };
+
+export const Group: Story = {
+  args: {
+    children: (
+      <>
+        <DropdownMenuTrigger asChild>
+          <Button
+            appearance="subtle"
+            icon={
+              <Avatar
+                name="Arya Stark"
+                size="sm"
+                src="https://images.unsplash.com/photo-1511485977113-f34c92461ad9?ixlib=rb-1.2.1&w=128&h=128&dpr=2&q=80"
+              />
+            }
+            iconAutosize
```

Comment:
can you rebase against `main` please? added a `AvatarTrigger` story that demonstrates how to use avatars inside buttons properly

got rid of `iconAutosize` for `iconOnly` prop and instead of using `icon` use children to pass the avatar